app/src/main/java/com/kelvin/bdaypost/birthday/add/AddBirthdayFragment.kt

```diff
@@ -0,0 +1,159 @@
+package com.kelvin.bdaypost.birthday.add
```

Comment:
maybe instead of having an `add` package inside of a `birthday` package, we can just have `addbirthday`, since most of the packages will be birthday-related anyway